x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -249,6 +249,8 @@ export const DETECTION_ENGINE_SIGNALS_MIGRATION_URL = `${DETECTION_ENGINE_SIGNAL
 export const DETECTION_ENGINE_SIGNALS_MIGRATION_STATUS_URL = `${DETECTION_ENGINE_SIGNALS_URL}/migration_status`;
 export const DETECTION_ENGINE_SIGNALS_FINALIZE_MIGRATION_URL = `${DETECTION_ENGINE_SIGNALS_URL}/finalize_migration`;
 
+export const RAC_ALERTS_BULK_UPDATE_URL = '/internal/rac/alerts/bulk_update';
```

Comment:
you might want to use this in place of this const here:

https://github.com/elastic/kibana/pull/108092/files#diff-72432ac56ed999450944dac918fcfc78995c78218ccdb5a8d5532eff769f63c6R14